internal/terraform/terraform.go

```diff
@@ -333,3 +289,71 @@ func OutputSingleValue(dryRun bool, directory, tfEntrypoint, outputName string)
 
 	log.Println("tfOutput is: ", tfOutput.String())
 }
+
+// ApplyUsersTerraform load environment variables into the host based on the git provider, change directory to the
+// Terraform required modules, terraform init, terraform apply and clean terraform files.
+// todo: break it into smaller functions with no dependencies in order to allow unit tests
+func ApplyUsersTerraform(dryRun bool, directory string, gitProvider string) error {
+
+	config := configs.ReadConfig()
+
+	if viper.GetBool("create.terraformapplied.users") || dryRun {
+		log.Println("skipping: ApplyUsersTerraform")
+		return nil
+	}
+
+	if len(gitProvider) == 0 {
+		return errors.New("git provider not provided, skipping terraform apply")
+	}
+
+	log.Println("Executing ApplyUsersTerraform")
+
+	//* AWS_SDK_LOAD_CONFIG=1
+	//* https://registry.terraform.io/providers/hashicorp/aws/2.34.0/docs#shared-credentials-file
+	envs := map[string]string{}
+
+	if gitProvider == "github" {
+		envs["GITHUB_TOKEN"] = os.Getenv("GITHUB_AUTH_TOKEN")
+		envs["GITHUB_OWNER"] = viper.GetString("github.owner")
+	} else if gitProvider == "gitlab" {
+		envs["GITLAB_TOKEN"] = viper.GetString("gitlab.token")
+		envs["TF_VAR_initial_password"] = viper.GetString("botpassword")
```

Comment:
We are setting this property 2x on this function: 
- envs["TF_VAR_initial_password"]
Lines: 320 and 332.. 

I think we want to keep only line `332` as both modes will support the bot password, I believe. @johndietz  can confirm on that.

Reply:
yep agreed it'll be the bot user on both git providers now and init password will be implemented on both sides